src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.type.ts

```diff
@@ -40,7 +40,7 @@ export type DidSignedIssuanceStatusArray = Static<typeof DidSignedIssuanceStatus
  * OCSP response
  */
 
-export const ValidOcspReasonCode = Number.withConstraint((n) => n >= 0 && n <= 10 && n != 7);
+export const ValidOcspReasonCode = Number.withConstraint((n) => [1, 2, 3, 4, 5, 6, 8, 9, 10, 1001].includes(n));
```

Comment:
I was just thinking, would this be simpler and still work?
```
export const ValidOcspReasonCode = Number.withConstraint((n) => Object.values(OcspResponderRevocationReason).includes(n));
```

This will need to import `OcspResponderRevocationReason` from `revocation.types.ts` into this file though. But what you have works too!

Reply:
Another thing is, `n >= 0 && n <= 10 && n != 7` includes `0` but `[1, 2, 3, 4, 5, 6, 8, 9, 10, 1001]` doesn't. So maybe need to add `0` if we prefer listing out the numbers.

Reply:
Good catch!
I will follow your suggestion as it is cleaner